lib/engine-object-file/src/builder.rs

```diff
@@ -76,13 +76,13 @@ mod tests {
     #[cfg(feature = "compiler")]
     use std::sync::Arc;
     #[cfg(feature = "compiler")]
-    use wasmer_compiler::{Compiler, FunctionMiddlewareGenerator};
+    use wasmer_compiler::{Compiler, ModuleMiddleware};
```

Comment:
This was actually not being tested, so I caught the bug when testing in all packages :)